python_modules/dagster/dagster/_core/definitions/data_time.py

```diff
@@ -429,7 +429,8 @@ def get_ignored_failure_data_time(
             return current_data_time
 
         run_failure_time = datetime.datetime.utcfromtimestamp(
-            latest_run_record.end_time or latest_run_record.create_timestamp.timestamp()
+            latest_run_record.end_time
+            or latest_run_record.create_timestamp.replace(tzinfo=datetime.timezone.utc).timestamp()
```

Comment:
can I humbly suggest replacing this with `datetime_as_float(latest_run_record.create_timestamp)` to outsource the creative timezone thinking to that function?

Reply:
(but thx for the quick fix)

Reply:
ah fair enough